### PR TITLE
PLT-1800 Load server side locale from the config.json

### PR DESCRIPTION
**Pull request by enahum**
There is a PR for the JS drivers that goes along with this one
https://github.com/mattermost/mattermost-driver-javascript/pull/1

Also added support for country specific locales and renamed `pt` to be used as `pt-BR`

### CODE REVIEW

File: model/config.go [hunk 1]
```diff
@@ -507,6 +514,16 @@ func (o *Config) SetDefaults() {
 		o.LdapSettings.NicknameAttribute = new(string)
 		*o.LdapSettings.NicknameAttribute = ""
 	}
+
+	if o.LocalizationSettings.DefaultServerLocale == nil {
+		o.LocalizationSettings.DefaultServerLocale = new(string)
+		*o.LocalizationSettings.DefaultServerLocale = DEFAULT_LOCALE
+	}
+
+	if o.LocalizationSettings.DefaultClientLocale == nil {
```
Comment (coreyhulen) [inline model/config.go]: No default for AvailableLocales?

Comment (enahum) [inline model/config.go]: holy!! yes it should be "en", adding that now!!

Comment (enahum) [inline model/config.go]: Change ready

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**